src/main/java/com/jalasoft/selenium/Ariel/movies/Customer.java

```diff
@@ -0,0 +1,83 @@
+package com.jalasoft.selenium.ariel.movies;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by ariel Mattos on 3/6/2017.
+ */
+
+class Customer {
+    private String name;
+    private List<Rental> rentals = new ArrayList<>();
+
+    /**
+     * Creates a Children movie with the given title.
+     * @param name          the name of the customer.
+     */
+    Customer(final String name) {
+        this.name = name;
+    }
+
+    /**
+     * Add a movie rental object to the list of the user's movie rentals.
+     * @param rental        a movie rental object.
+     */
+    public void addRental(final Rental rental) {
+        rentals.add(rental);
+    }
+
+    /**
+     * Returns the name of the user.
+     * @return              a String containing the name of the user.
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Returns a statement of the movie rentals, total charges and frequent renter points.
+     * @return               a string containing the movie rentals, total charges and frequent renter points.
+     */
+    public String statement() {
+        StringBuilder report = new StringBuilder();
+        report.append("Rental Record for ").append(getName()).append("\n");
```

Comment:
you can use directly the attribute name here instead of calling to getName() method